Website/Portals/_default/HotcakesViews/_default/Scripts/StripeCheckout.js

```diff
@@ -0,0 +1,116 @@
+// This is a public sample test API key.
+// Don’t submit any personally identifiable information in requests made with this key.
+// Sign in to see your own test API key embedded in code samples.
+const stripe = Stripe("pk_test_TYooMQauvdEDq54NiTphI7jx");
+
+// The items the customer wants to buy
+const items = [{ id: "xl-tshirt" }];
+
+let elements;
+
+initialize();
+checkStatus();
+
+document
+  .querySelector("#Form")
+  .addEventListener("submit", handleSubmit);
+
+// Fetches a payment intent and captures the client secret
+async function initialize() {
+    const response = await fetch("/DesktopModules/Hotcakes/API/mvc/PaymentIntentApi/CreatePaymentIntent", {
+    method: "POST",
+    headers: { "Content-Type": "application/json" },
+    body: JSON.stringify({ items }),
+  });
+  const { clientSecret } = await response.json();
+
+  const appearance = {
+    theme: 'stripe',
+  };
+  elements = stripe.elements({ appearance, clientSecret });
+
+  const paymentElement = elements.create("payment");
+  paymentElement.mount("#payment-element");
+}
+
+async function handleSubmit(e) {
+  e.preventDefault();
+  setLoading(true);
+
+  const { error } = await stripe.confirmPayment({
+    elements,
+    confirmParams: {
+      // Make sure to change this to your payment completion page
+      return_url: "http://localhost:4242/checkout.html",
```

Comment:
This should be a different value entirely.  Check the setup instructions in the documentation for this kind of URL value.  Specifically, look at how the current PayPal Express and other payment methods handle the return URL.  

Though, this use case is a tiny bit different.  It may take some testing and debugging to get the correct value - especially since you'll never know what the domain name is.